tests/test_collection_crs.py

```diff
@@ -27,10 +29,35 @@ def test_collection_no_crs_wkt(tmpdir, path_coutwildrnp_shp):
 @requires_gdal2
 def test_collection_create_crs_wkt(tmpdir):
     """A collection can be created using crs_wkt"""
-    filename = str(tmpdir.join("test.shp"))
+    filename = str(tmpdir.join("test.geojson"))
     wkt = 'GEOGCS["GCS_WGS_1984",DATUM["WGS_1984",SPHEROID["WGS_84",6378137,298.257223563]],PRIMEM["Greenwich",0],UNIT["Degree",0.017453292519943295],AUTHORITY["EPSG","4326"]]'
     with fiona.open(filename, 'w', schema={'geometry': 'Point', 'properties': {'foo': 'int'}}, crs_wkt=wkt, driver='GeoJSON') as dst:
-        assert dst.crs_wkt == wkt
+        assert dst.crs_wkt.startswith('GEOGCS["WGS 84')
+
+    with fiona.open(filename) as col:
+        assert col.crs_wkt.startswith('GEOGCS["WGS 84')
```

Comment:
```suggestion
        assert col.crs_wkt.startswith('GEOGCS["GCS_WGS_1984')
```

See https://travis-ci.org/Toblerity/Fiona/jobs/608201443#L1157.

Reply:
Ha, passes on GDAL 1 & 3, but not 2. 

Thoughts on:
```python
assert col.crs_wkt.startswith('GEOGCS["WGS 84') or col.crs_wkt.startswith('GEOGCS["GCS_WGS_1984')
```

Reply:
No, the test is skipped with GDAL 1.11 and GDAL 3.0.

Reply:
Oh ha, it skips GDAL 1.1. But, I don't think it skips 3.0:
```python
requires_gdal2 = pytest.mark.skipif(
    not gdal_version.major >= 2,
    reason="Requires GDAL 2.x")
```
> Since those WKT are equivalent, I think that's a good approach.

Sounds like a good plan to me :+1:

Reply:
Fix pushed.